utils/utils.go

```diff
@@ -144,14 +144,26 @@ func StartReaper() {
 		for {
 			// Wait for a child to terminate
 			sig := <-sigs
-			for {
+			for sig == syscall.SIGCHLD {
```

Comment:
don't we drop all other sigs this way? should we have a switch inside the for loop